components/brave_rewards/common/pref_names.cc

```diff
@@ -23,8 +23,6 @@ const char kBackupNotificationInterval[] =
 const char kBackupSucceeded[] = "brave.rewards.backup_succeeded";
 const char kUserHasFunded[] = "brave.rewards.user_has_funded";
 const char kUserHasClaimedGrant[] = "brave.rewards.user_has_claimed_grant";
-const char kAddFundsNotification[] =
```

Comment:
AFAIK, there's no way to get rid of a deprecated pref in the `Preferences` file, as in order to be able to remove it, you'd have to register it. So I think it'd be better to just mark it as `DEPRECATED` in the header file to make sure we don't encounter any weirdness, in case we accidentally reuse the same path in the future.

Reply:
Ah yes, good point. Fixed.